pages/docs/manual/latest/build-configuration.mdx

```diff
@@ -151,16 +153,15 @@ one of the following:
 - `".bs.mjs"`
 - `".bs.cjs"`
```

Comment:
It would be nice to discourage `.bs` suffix

Reply:
I just don't mention it anymore.